contracts/EntryPoint.sol

```diff
@@ -83,6 +83,27 @@ contract EntryPoint is StakeManager {
         require(success);
     }
 
+    /**
+     * execute a user op
+     * @param opIndex into into the opInfo array
+     * @param userOp the userOp to execute
+     * @param opInfo the opInfo filled by validatePrepayment for this userOp.
+     * @return collected the total amount this userOp paid.
+     */
+    function _executeUserOp(uint256 opIndex, UserOperation calldata userOp, UserOpInfo memory opInfo) private returns (uint256 collected) {
+        uint256 preGas = gasleft();
+        uint256 contextOffset = opInfo.contextOffset;
+        bytes memory context;
+        assembly {context := contextOffset}
```

Comment:
Can we convert these assembly pieces into utility functions of
`function getUintMemoryPointer(bytes memory) private view returns (uint)`
and
`function getBytesFromMemoryPointer(uint point) private view returns (bytes memory)`
?
Like, Solidity won't copy the `bytes memory` to pass it into a function, right?

Not that it is necessary, just weird one-liner assembly insertions in the middle of regular function seems weird.